src/VisualStudio/Core/Test.Next/Services/PerformanceTrackerServiceTests.cs

```diff
@@ -131,7 +133,7 @@ private IEnumerable<AnalyzerPerformanceInfo> CreateSnapshots(Dictionary<string,
                 for (var j = 0; j < data.Length; j++)
                 {
                     double result;
-                    if (!double.TryParse(data[j], out result))
+                    if (!double.TryParse(data[j], NumberStyles.Float | NumberStyles.AllowThousands, EnsureEnglishUICulture.PreferredOrNull, out result))
```

Comment:
@heejaechang @sharwell @ivanbasov Are the files guaranteed to be in the English locale during generation?

Reply:
Know the IDE team typically prefers current culture over the compiler preference of forcing English. In this case though the CSV files are checked into the code base and exposed as an embedded resource. Hence we need to pick a fixed culture to use during parse here as they copy will always be in a single locale.